src/main/resources/templates/clojure-lsp/clientSettings.json

```diff
@@ -1,5 +1,8 @@
 {
   "completions": {
     "caseSensitive": true
+  },
+  "workspaceSymbol": {
```

Comment:
Added default for all bundled templates with the feature enabled for languages that have the notion of a type and disabled for those that don't.